src/nodes.coffee

```diff
@@ -102,6 +136,54 @@ exports.Base = class Base
         parts.push    @makeCode ")"
     parts
 
+  compileCommentFragments: (o, node, fragments) ->
+    return fragments unless node.comments
+    # This is where comments, that are attached to nodes as a `comments`
+    # property, become `CodeFragment`s. “Inline block comments,” e.g.
```

Comment:
The CoffeeScript docs are written in U.S. English: https://english.stackexchange.com/a/114403.